lib/solr_wrapper/instance.rb

```diff
@@ -222,11 +218,10 @@ def with_collection(options = {})
     end
 
     ##
-    # Clean up any files solr_wrapper may have downloaded
+    # Clean up any files solr_wrapper may have downloaded, except the solr-{ver}.zip file
     def clean!
       stop
       remove_instance_dir!
-      FileUtils.remove_entry(config.download_dir, true) if File.exist?(config.download_dir)
```

Comment:
`clean` vs. `clean!` maybe?  where the latter calls this `clean` and then also cleans up the downloads (all of them).